browser-test/src/admin/admin_program_creation.test.ts

```diff
@@ -40,6 +40,64 @@ test.describe('program creation', () => {
     await adminProgramImage.expectProgramImagePage()
   })
 
+  test('create program with disabled visibility condition feature enabled', async () => {
+    const {page, adminPrograms, adminProgramImage} = ctx
+    await enableFeatureFlag(page, 'disabled_visibility_condition_enabled')
+    await loginAsAdmin(page)
+
+    await adminPrograms.addProgram(
+      'program name',
+      'description',
+      'https://usa.gov',
+      ProgramVisibility.DISABLED,
+      'admin description',
```

Comment:
nitpick (non-blocking): The params from here to the bottom all have default values so you could leave them off.

Reply:
With default values, it leads this program being submitted. To test the visibility options containing `Disabled`, we need to stay on the program editing page.

Reply:
Thank you for explaining!